src/_variables.scss

```diff
@@ -139,7 +139,7 @@ $yiq-text-light:            $white !default;
 
 $enable-caret:              true !default;
 $enable-rounded:            true !default;
-$enable-shadows:            false !default;
+$enable-shadows:            true;
```

Comment:
I assume this affects more than just popovers. Are we going to need to override this back to false for some components?